vulture_os/services/frontend/models.py

```diff
@@ -1599,6 +1599,21 @@ class Frontend(RsyslogQueue, models.Model):
         help_text=_("Perception Point X Ray API Key"),
         default="",
     )
+    perception_point_x_ray_organization_id = models.TextField(
+        verbose_name=_("Perception Point X Ray Organization ID"),
+        help_text=_("Perception Point X Ray Organization ID"),
+        default="",
+    )
+    perception_point_x_ray_environment_id = models.TextField(
+        verbose_name=_("Perception Point X Ray Environment ID"),
+        help_text=_("Perception Point X Ray Environment ID"),
+        default="",
+    )
+    perception_point_x_ray_case_types = models.TextField(
+        verbose_name=_("Perception Point X Ray Case Types"),
+        help_text=_("Perception Point X Ray Case Types"),
+        default="",
+    )
```

Comment:
Shouldn't this be a list of (Positive) Integers?
(if so, use a JSONField and validate the values in the form)